private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -186,6 +185,8 @@ export class TemplateServiceClient {
     if (!result.success){
       return { success: false, errorMessage: ServiceErrorMessage.UserNotFound }
     }
+    logger.info(`User with oid ${authId} requested data.`);
```

Comment:
Do we need to log what data was requested and returned?

Reply:
Looks like we don't:
![image](https://user-images.githubusercontent.com/21356912/76988260-ae659c80-6901-11ea-8014-c9168b860fe0.png)